app/models/filter.rb

```diff
@@ -135,6 +141,21 @@ def expire_topbar_cache(sweeper)
     role.usergroups.each { |g| g.expire_topbar_cache(sweeper) }
   end
 
+  def disable_overriding!
+    self.override = false
+    self.save!
+  end
+
+  def inherit_taxonomies
+    inherit_taxonomies! unless self.override?
```

Comment:
The method itself is useful from outside context, currently only used in tests directly but this endpoint should be present IMO. The unless belongs into the method, it only makes sense to inherit if filter does not override taxonomies. If we put it into before_save and someone else calls this method he should add this condition explicitly and basically duplicate the logic.

I don't want to start another rubocop discussion so I only add I don't like `=> -> {...}` syntax too.